SwiftUIProject/SwiftUIProject/Profile/ProfileViewModel.swift

```diff
@@ -0,0 +1,63 @@
+// ProfileViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
```

Comment:
```suggestion
import SwiftUI
```
тут используется SUI?